src/swarmRegistryAdapter.js

```diff
@@ -0,0 +1,52 @@
+// Run this first:
```

Comment:
This probably belongs in the `README.md` instead of this file (though this file should note that it expects ganache to be running). Even better, could the automation do this as part of setup?

Reply:
Yep definitely, will add as part of final integration / regression testing refactor with the latest and greatest